tests/admin_scripts/tests.py

```diff
@@ -118,17 +119,26 @@ def run_test(self, args, settings_file=None, apps=None):
         test_environ['PYTHONPATH'] = os.pathsep.join(python_path)
         test_environ['PYTHONWARNINGS'] = ''
 
+        additional_kwargs = dict()
```

Comment:
I would rename it to `kwargs`
```suggestion
        kwargs = {}
```